model_checking/sail_prelude/Cargo.toml

```diff
@@ -7,3 +7,4 @@ edition = "2021"
 path = "lib.rs"
 
 [dependencies]
+rand = "0.9.0-beta.1"
```

Comment:
We most likely don't want `rand` here.

Reply:
In the prelude there is no randomness, it is only used for the unit tests.